### PR TITLE
added ccl samples for bs and df

### PR DESCRIPTION
**Pull request by makart19**
- added ccl samples for bs and df
- made formatting for corresponding mpi samples

### CODE REVIEW

File: samples/oneapi/dpc/ccl/sources/basic_statistics_distr_ccl.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    auto comm =
+        dal::preview::spmd::make_communicator<dal::preview::spmd::backend::ccl>(
+            queue);
+    auto rank_id = comm.get_rank();
+    auto rank_count = comm.get_rank_count();
+
+    auto input_vec = split_table_by_rows<float>(queue, data, rank_count);
+
+    const auto result = dal::preview::compute(comm, bs_desc, input_vec[rank_id]);
+    if (comm.get_rank() == 0) {
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/basic_statistics_distr_ccl.cpp:50]:
```suggestion
    if (rank_id == 0) {
```

File: samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        std::cout << "Variable importance results:\n"
+                  << result_train.get_var_importance() << std::endl;
+
+        std::cout << "OOB error: " << result_train.get_oob_err() << std::endl;
+    }
+
+    const auto result_infer = dal::preview::infer(
+        comm, df_desc, result_train.get_model(), x_test_vec[rank_id]);
+
+    if (comm.get_rank() == 0) {
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp:92]:
```suggestion
    if (rank_id == 0) {
```

File: samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            .set_error_metric_mode(df::error_metric_mode::out_of_bag_error)
+            .set_variable_importance_mode(df::variable_importance_mode::mdi)
+            .set_infer_mode(df::infer_mode::class_responses |
+                            df::infer_mode::class_probabilities)
+            .set_voting_mode(df::voting_mode::weighted);
+
+    const auto result_train = dal::preview::train(
+        comm, df_desc, x_train_vec[rank_id], y_train_vec[rank_id]);
+
+    if (comm.get_rank() == 0) {
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp:82]:
```suggestion
     if (rank_id == 0) {
```

File: samples/oneapi/dpc/mpi/sources/basic_statistics_distr_mpi.cpp [hunk 4]
```diff
@@ -17,68 +17,68 @@
...
-    std::cout << "Mean:\n" << result.get_mean() << std::endl;
-    std::cout << "Second order raw moment:\n"
-              << result.get_second_order_raw_moment() << std::endl;
-    std::cout << "Variance:\n" << result.get_variance() << std::endl;
-    std::cout << "Standard deviation:\n"
-              << result.get_standard_deviation() << std::endl;
-    std::cout << "Variation:\n" << result.get_variation() << std::endl;
-  }
+    const auto result = dal::preview::compute(comm, bs_desc, input_vec[rank_id]);
+    if (comm.get_rank() == 0) {
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/mpi/sources/basic_statistics_distr_mpi.cpp:50]:
```suggestion
     if (rank_id == 0) {
```

File: samples/oneapi/dpc/mpi/sources/decision_forest_cls_hist_distr_mpi.cpp [hunk 5]
```diff
@@ -17,106 +17,103 @@
...
+            .set_error_metric_mode(df::error_metric_mode::out_of_bag_error)
+            .set_variable_importance_mode(df::variable_importance_mode::mdi)
+            .set_infer_mode(df::infer_mode::class_responses |
+                            df::infer_mode::class_probabilities)
+            .set_voting_mode(df::voting_mode::weighted);
+
+    const auto result_train = dal::preview::train(
+        comm, df_desc, x_train_vec[rank_id], y_train_vec[rank_id]);
+
+    if (comm.get_rank() == 0) {
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/mpi/sources/decision_forest_cls_hist_distr_mpi.cpp:82]:
```suggestion
     if (rank_id == 0) {
```

File: samples/oneapi/dpc/ccl/sources/decision_forest_reg_hist_distr_ccl.cpp [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+        std::cout << "OOB error: " << result_train.get_oob_err() << std::endl;
+        std::cout << "OOB error per observation:\n"
+                  << result_train.get_oob_err_per_observation() << std::endl;
+    }
+
+    const auto result_infer = dal::preview::infer(
+        comm, df_desc, result_train.get_model(), x_test_vec[rank_id]);
+
+    if (comm.get_rank() == 0) {
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/decision_forest_reg_hist_distr_ccl.cpp:88]:
```suggestion
     if (rank_id == 0) {
```

Comment (makart19) [reply samples/oneapi/dpc/ccl/sources/decision_forest_reg_hist_distr_ccl.cpp:88]: changed all conditions to if (comm.get_rank() == 0) { to get it in one style

File: samples/oneapi/dpc/mpi/sources/decision_forest_cls_hist_distr_mpi.cpp (hunk 5 above)
Comment (makart19) [reply samples/oneapi/dpc/mpi/sources/decision_forest_cls_hist_distr_mpi.cpp:82]: changed all conditions to if (comm.get_rank() == 0) { to get it in one style

File: samples/oneapi/dpc/ccl/sources/basic_statistics_distr_ccl.cpp (hunk 1 above)
Comment (makart19) [reply samples/oneapi/dpc/ccl/sources/basic_statistics_distr_ccl.cpp:50]: changed all conditions to if (comm.get_rank() == 0) { to get it in one style

**Review by amgrigoriev — APPROVED** [review]

File: samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp (hunk 3 above)
Comment (makart19) [reply samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp:82]: changed all conditions to if (comm.get_rank() == 0) { to get it in one style

File: samples/oneapi/dpc/mpi/sources/basic_statistics_distr_mpi.cpp (hunk 4 above)
Comment (makart19) [reply samples/oneapi/dpc/mpi/sources/basic_statistics_distr_mpi.cpp:50]: changed all conditions to if (comm.get_rank() == 0) { to get it in one style

File: samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp (hunk 2 above)
Comment (makart19) [reply samples/oneapi/dpc/ccl/sources/decision_forest_cls_hist_distr_ccl.cpp:92]: changed all conditions to if (comm.get_rank() == 0) { to get it in one style

**Review by Alexandr-Solovev — APPROVED** [review]

**Pull request merged**